spec/library_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'yaml'
+require './lib/library.rb'
+
+describe Library do
+  it 'should have the list of books' do
+    expect(subject.bookrack).not_to be nil
+  end
+
+  it 'should check if the book is available' do
+     expect(subject.bookrack[0][:available]).to be true
+  end
+
+  it 'should check if the book is not available' do
+     expect(subject.bookrack[2][:available]).to be false
+  end
```

Comment:
Ok, but we should have a method in our implementation file `library.rb` that can check if a book is available. Then we test it here. This test is only checking the data itself, not how we use the data in our program.